route/build/src/write_simoutput.f90

```diff
@@ -213,7 +213,8 @@ subroutine defineStateFile(fname,           &  ! input: filename
  ! define time bound 
  call defvar('time_bound', 'time bound at last time step',                'sec', (/sTB_DimName/),                               nf90_double, ierr, cmessage)
  ! Hill-Slope 
- call defvar('QFUTURE',    'Hill slope convoluted flow',                  'm3/s',(/sSeg_DimName,sTdh_DimName,sEns_dimName/),    nf90_double, ierr, cmessage)
+ call defvar('QFUTURE',    'Hill slope routed future flow',    'm3/s',(/sSeg_DimName,sTdh_DimName,sEns_dimName/),    nf90_double, ierr, cmessage)
```

Comment:
changes look fine but it would be helpful to add a comment to say what 'future flow' and 'delayed runoff' are in the context of the routing.